sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -85,6 +85,30 @@ object ScalaJSPlugin extends AutoPlugin {
     }
 
     /**
+     *  Creates a [[sbt.Def.Initialize Def.Initialize]] for a JSDOMNodeJSEnv. Use
+     *  this to explicitly specify in your build that you would like to run with
+     *  Node.js on a JSDOM window:
+     *
+     *  {{{
+     *  jsEnv := JSDOMNodeJSEnv().value
+     *  }}}
+     *
+     *  Note that the resulting [[sbt.Def.Setting Setting]] is not scoped at
+     *  all, but must be scoped in a project that has the ScalaJSPlugin enabled
+     *  to work properly.
+     *  Therefore, either put the upper line in your project settings (common
+     *  case) or scope it manually, using
+     *  [[sbt.ProjectExtra.inScope[* Project.inScope]].
```

Comment:
Actually all links to sbt `0.13.9` are broken, it looks like they are meant for sbt `0.12`. The links is resoved as: 
```
http://www.scala-sbt.org/0.13.9/api/index.html#sbt.ProjectExtra@inScope[T](scope:<?>,i:<?>):sbt.Def.Initialize[T]
```
and does not work.

But changing `0.13.9` to `0.12.1` makes the link work:
```
http://www.scala-sbt.org/0.12.1/api/index.html#sbt.ProjectExtra@inScope[T](scope:<?>,i:<?>):sbt.Def.Initialize[T]
```